tests/test_utils.py

```diff
@@ -132,3 +134,23 @@ async def mutate_registrations():
     assert cb2.exec_count == 1  # Unregistered by previous invoke, not called again
     assert cb3.exec_count == 1  # once=True, so not called again
     assert cb4.exec_count == 1  # Registered during previous invoke(), was called
+
+
+def test_random_port():
+    assert random_port(9000, 9000) == 9000
+
+    seen: Set[int] = set()
+    while len(seen) < 10:
+        seen.add(random_port(9001, 9010))
```

Comment:
Did you intend to `assert` something about `seen` here?

Reply:
No but I forgot to add a comment explaining the intent. Thanks!